codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
```

Comment:
This file could be moved to `.github` folder in order not to pollute the main directory with too many config files

Reply:
Done 👍